server/controllers/work/addWork.js

```diff
@@ -0,0 +1,25 @@
+const { workValidation, uploadImage } = require('../../utils');
+const { customError } = require('../errors');
+const { Work } = require('../../database/models');
+
+const addWork = async (request, response, next) => {
+  try {
+    const { providerID } = request;
+    const { image } = request.files;
+    const { title, content } = await workValidation.validateAsync(request.body);
```

Comment:
try to also validate providerID and image
```suggestion
    const { title, content } = await workValidation.validateAsync({...request.body, providerID, image});
```